enumcover.go

```diff
@@ -23,6 +23,11 @@ var Analyzer = &analysis.Analyzer{
 }
 var commentRegex = regexp.MustCompile(`enumcover:([\w\.]+)`)
 
+// Keep a map of renamed constants, where key => old name and value => new name
+var renamedConstants = map[string]string{
+	"Ptr": "Pointer",
+}
```

Comment:
Hmm, can we work out this kind of thing by code inspection instead? Like we should be able to know that Ptr = Pointer because of the alias right here: https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/reflect/type.go;l=272

Otherwise we'd need to update this map for every type we want to support that has an alias, which limits this package's usefulness to types it already knows about.

Reply:
Okay, thanks for the feedback. I will try and fix it tomorrow.